src/pudl/metadata/resources/eia861.py

```diff
@@ -694,6 +694,47 @@
         "sources": ["eia861"],
         "etl_group": "eia861",
     },
+    "core_eia861__yearly_short_form": {
+        "description": (
+            """This is data extracted from Form EIA-861S, a shorter version of Form EIA-861.
+The data started being reported in 2012. However, the 2019 data is not available.
```

Comment:
I think we should clarify that the 2019 data was reported intermingled with the long-form data in all of the other tables, and hasn't yet been disentangled from them.